src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styles from './Pagination.module.sass'
+import { useDispatch, useSelector } from 'react-redux'
+import { setFilter, setCurrentPage } from '../../store/slices/ProductSlice'
+import { useTranslation } from 'react-i18next';
+
+function Pagination() {
+
+    const { t } = useTranslation()
+
+    const dispatch = useDispatch()
+
+    const pagination = useSelector( state => state.products.pagination)
+    const filter = useSelector( state => state.products.filter)
+    const currentPage = useSelector( state => state.products.currentPage)
+    const { _limit, _totalRows } = pagination
+    const totalPages = Math.ceil(_totalRows / _limit);
+
+    function handleRenderPagination() {
+        const totalPagesArr = Array.from({ length: totalPages }, (_, i) => i + 1);
+        const renderBigArray =
+            totalPages <= 8
+                ? totalPagesArr
+                : [currentPage, currentPage + 1, currentPage + 2, "...", totalPages];
+        return renderBigArray.map((item) => {
+            return (
+                <button
+                    onClick={() => handlePageChange(item)}
+                    className={item === currentPage ? styles.active : "non-active"}
+                    key={item}
+                    >
+                    {item}
+                </button>
+            );
+        });
+    }
+
+    function handlePageChange(newPage) {
+        dispatch(setCurrentPage(newPage));
+        dispatch(setFilter({
+            ...filter,
+            _page: newPage
+        }))
+    }
+
+    return (
+        <div>
+            <div className={styles.pagination}>
```

Comment:
```suggestion
            <div className={styles.pagination}>
```
Dư 1 thẻ bọc ngoài nhỉ?